graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/GqlJcrPropertyI18nInput.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2002-2022 Jahia Solutions Group SA. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jahia.modules.graphql.provider.dxm.node;
+
+import graphql.annotations.annotationTypes.GraphQLDescription;
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLName;
+import graphql.annotations.annotationTypes.GraphQLNonNull;
+
+import java.util.List;
+
+@GraphQLName("JCRPropertyI18n")
+@GraphQLDescription("GraphQL representation of a JCR property to set")
+public class GqlJcrPropertyI18nInput {
```

Comment:
Is it necessary to create a new class here to perform the op? Would the original GqlJcrPropertyInput would also work here if we just provide name and language only?

The class and the hierarchy is a bit strange to me as having a property class that doesn't carry at least a name/value. I also expect i18n as a specialized version of the original and the hierarchy would be the other way around, with the i18n class adding support for language.

If it's needed, we can maybe rename as something else (e.g. GqlJcrPropertyFetcherInput) so as it's not confused with a property input that doesn't support values or any of the other property params.

Reply:
I agree that this and `GqlJcrPropertyInput` could be either combined or tied together via inheritance

Reply:
ok, agreed with the point of showing options that do not matter for the op, so yeah maybe a rename might be better, just to also avoid confusion that it's not a full property input object.